bootstrap/bootstrap.go

```diff
@@ -52,67 +54,62 @@ type Bootstrap struct {
 }
 
 func New(config *config.Config) (*Bootstrap, error) {
-	logger := zerolog.New(config.LogWriter).With().Timestamp().Logger()
-	logger = logger.Level(config.LogLevel)
-	logger.Info().Msg("starting up the EVM gateway")
+	logger := zerolog.New(config.LogWriter).
+		With().Timestamp().Logger().Level(config.LogLevel)
 
-	// create pebble storage from the provided database root directory
-	store, err := pebble.New(config.DatabaseDir, logger)
+	client, err := setupCrossSporkClient(config, logger)
 	if err != nil {
 		return nil, err
 	}
 
-	// create access client with cross-spork capabilities
-	currentSporkClient, err := grpc.NewClient(config.AccessNodeHost)
-	if err != nil {
-		return nil, fmt.Errorf("failed to create client connection for host: %s, with error: %w", config.AccessNodeHost, err)
-	}
-
-	// if we provided access node previous spork hosts add them to the client
-	pastSporkClients := make([]access.Client, len(config.AccessNodePreviousSporkHosts))
-	for i, host := range config.AccessNodePreviousSporkHosts {
-		grpcClient, err := grpc.NewClient(host)
-		if err != nil {
-			return nil, fmt.Errorf("failed to create client connection for host: %s, with error: %w", host, err)
-		}
-
-		pastSporkClients[i] = grpcClient
-	}
-
-	// initialize cross spork client to the access nodes
-	client, err := requester.NewCrossSporkClient(
-		currentSporkClient,
-		pastSporkClients,
-		logger,
-		config.FlowNetworkID,
-	)
+	storages, err := setupStorage(config, logger)
```

Comment:
Although we have just setup the cross-spork client above:
```go
client, err := setupCrossSporkClient(config, logger)
```
The `setupStorage` function seems to do the same thing again in: https://github.com/onflow/flow-evm-gateway/pull/481/files/f3dd6ea243e25f5f750ac38f783b217e88968919..5fcf97bbd7e52367d4b9a5a32bb9166e13774237#diff-70e82e7432e3056b7466bb3e4fe16874c12cfbe7bbb71a15ad9d1997217e94c9R402-R405
Could we maybe just pass in the created `client`?

Reply:
this was an oversight, thank you